### ISSUE TITLE
[PropertyGrid] Missing CSS class k-grid-aria-root

### ISSUE THREAD

**Issue by NickIliev** [telerik/kendo-ui-core#8475]
The **PropertyGrid** in **Kendo JQuery** component has a rendering mismatch when compared to the source of truth tests:

[https://github.com/telerik/kendo-themes/blob/develop/tests/propertygrid/propertygrid.html#L76C33-L76C49](https://github.com/telerik/kendo-themes/blob/develop/tests/propertygrid/propertygrid.html#L76C33-L76C49)

The ` k-grid-aria-root` is missing from the produced rendering

**Steps to reproduce:**

1. Open the PropertyGrid demo at [https://demos.telerik.com/kendo-ui/propertygrid/index](https://demos.telerik.com/kendo-ui/propertygrid/index)
2. Observe the produced rendering:

**Result**

The `k-grid-aria-root` is missing, which causes a styling mismatched with a theme generated from ThemeBuilder.

Reported through t.1714747
